legend-engine-language-pure-dsl-service-execution/src/main/java/org/finos/legend/engine/language/pure/dsl/service/execution/ServiceRunnerBuilder.java

```diff
@@ -0,0 +1,117 @@
+package org.finos.legend.engine.language.pure.dsl.service.execution;
+
+import org.eclipse.collections.api.list.MutableList;
+import org.eclipse.collections.impl.list.mutable.FastList;
+import org.eclipse.collections.impl.utility.Iterate;
+import org.finos.legend.engine.plan.execution.PlanExecutor;
+import org.finos.legend.engine.plan.execution.stores.StoreExecutor;
+import org.finos.legend.engine.plan.execution.stores.StoreExecutorBuilder;
+import org.finos.legend.engine.plan.execution.stores.StoreExecutorConfiguration;
+import org.finos.legend.engine.plan.execution.stores.inMemory.plugin.InMemoryStoreExecutorBuilder;
+import org.finos.legend.engine.plan.execution.stores.relational.config.RelationalExecutionConfiguration;
+import org.finos.legend.engine.plan.execution.stores.relational.config.TemporaryTestDbConfiguration;
+import org.finos.legend.engine.plan.execution.stores.relational.plugin.RelationalStoreExecutorBuilder;
+import org.finos.legend.engine.plan.execution.stores.service.plugin.ServiceStoreExecutorBuilder;
+
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.stream.Collectors;
+
+public class ServiceRunnerBuilder
+{
+    private TemporaryTestDbConfiguration temporaryTestDbConfiguration;
+    private RelationalExecutionConfiguration relationalExecutionConfiguration;
+    private String serviceRunnerClass;
+    private boolean allowJavaCompilation;
+
+    private ServiceRunnerBuilder()
+    {
+    }
+
+    public static ServiceRunnerBuilder newInstance()
+    {
+        return new ServiceRunnerBuilder();
+    }
+
+    public ServiceRunnerBuilder withServiceRunnerClass(String serviceRunnerClass)
+    {
+        this.serviceRunnerClass = serviceRunnerClass;
+        return this;
+    }
+
+    public ServiceRunnerBuilder withRelationalExecutorConfiguration(RelationalExecutionConfiguration relationalExecutionConfiguration)
```

Comment:
As discussed, the method can be withStoreExecutorConfigurations(StoreExecutorConfiguration... configs) to avoid compile time dependency on store modules

Reply:
Done.